requirements.txt

```diff
@@ -1,15 +1,24 @@
-numpy==1.15.2
+numpy==1.16.4
 matplotlib==3.0.3
-pandas==0.24.2
+pandas==0.25.1
 scipy==1.3.0
 joblib==0.13.2
+jupyter==1.0.0
 keras==2.2.4
 keras-applications==1.0.8
 keras-preprocessing==1.1.0
+plotly==3.10.0
 scikit-learn==0.21.3
+seaborn==0.9.0
+scipy==1.3.0
 tensorboard==1.14.0
 tensorflow==1.14.0
 tensorflow-estimator==1.14.0
 microsoft-bonsai-api==0.1.2
-bonsai-cli==1.0.1
-vpython==7.6.0
\ No newline at end of file
+bonsai-cli==1.0.4
+vpython==7.6.0
+cufflinks==0.16
+joblib==0.13.2
+pyyaml==5.3.1
+h5py==2.10.0
```

Comment:
good catch on the h5py, i did not include this as an oversight from my previous bugfix.